### ISSUE TITLE
[1.11.2] Storagedrawers and Mekanism crash

### ISSUE THREAD

**Issue by marcin212** [jaquadro/StorageDrawers#492]
https://github.com/aidancbrady/Mekanism/issues/4407

**Comment by jaquadro** [comment]
So assuming you don't somehow have a corrupt copy of the Storage Drawers jar, this is probably a mod conflict.

Another mod in your list must have packed in an older version of the Storage Drawers API, without including all the package-info stuff.  Try testing with just Storage Drawers, Mekanism, and their dependencies to check.

**Comment by marcin212** [comment]
I run this script:
```
for file in *.jar; do
    if ( unzip -c "$file" | grep -q "jaquadro"); then
        echo "$file"
    fi
done
```
Result:
Chameleon-1.11.2-3.1.0.jar
refinedstorage-1.4.10.jar
StorageDrawers-1.11.2-4.2.7.jar

**Comment by jaquadro** [comment]
Did you test temporarily removing RS?  Did it make any difference?

I checked and they do pack in an outdated version of the API.  However it doesn't look like they're doing anything wrong (includes package-info), but maybe their version is still 'winning' for some reason?

**Comment by marcin212** [comment]
I will test it later without RS.
Mod Jar : https://cloud.bymarcin.com/s/n9ZPFeDIZowKnm2
MD5      : 5C40628C2493476631F503E75ED56936
SHA1    : EE7997DDAAAE55ECD9341E68AC1393B53016A26D

**Comment by jaquadro** [comment]
Jar looks fine, so yeah, try without RS to confirm, then we can try to figure out what's broken with API resolution.

**Comment by xannor** [comment]
I just ran across this issue, and it is not Mekanism, I also have RS installed so I opened the jar and axed the SD api (going by memory from old RF api issues.) and both my server and client could launch and so far everything seems ok.

Strangley enough this did not occur until I shift right clicked on a draw controller with a forestry mining bag. 

This is with RS 1.4.14 so I think they need to update their API's.
